pybloqs/block/collapsible.py

```diff
@@ -0,0 +1,68 @@
+from typing import ClassVar
+
+from pybloqs.block.base import BaseBlock
+from pybloqs.block.layout import CompositeBlockMixin
+from pybloqs.html import append_to
+from pybloqs.static import Css
+from pybloqs.util import Cfg
+
+
+class CollapsibleBlock(CompositeBlockMixin, BaseBlock):
+    """A block that can be collapsed/expanded in HTML view.
+
+    Parameters
+    ----------
+    contents : list, tuple or set
+        Elements to be included in the collapsible block.
+    cascade_cfg : bool, default=True
+        If True, enables parameter cascading from this block.
+        If False, child blocks do not inherit parameters.
+    **kwargs : dict
+        Optional styling arguments. The `style` keyword has special meaning
+        for grouping styling parameters.
+
+    Notes
+    -----
+    The `style` keyword argument is useful when styling parameter names
+    clash with standard block parameters.
+    """
+
+    resource_deps: ClassVar[list] = [

```

Comment:
This should [actually be a tuple](https://github.com/man-group/PyBloqs/blob/master/pybloqs/block/base.py#L27)